pkg/proxy/proxy.go

```diff
@@ -364,73 +361,16 @@ func (p *Proxy) processEndpointResponse(webhookEvent *websocket.Attempt, resp *h
 //
 
 // New creates a new Proxy
-func New(cfg *Config, source *hookdecksdk.Source, connections []*hookdecksdk.Connection) *Proxy {
+func New(cfg *Config, connections []*hookdecksdk.Connection) *Proxy {
 	if cfg.Log == nil {
 		cfg.Log = &log.Logger{Out: ioutil.Discard}
 	}
 
-	connections_paths := make(map[string]string)
-
-	for _, connection := range connections {
-		connections_paths[connection.Id] = *connection.Destination.CliPath
-	}
```

Comment:
@alexbouchardd do you know how this is used? I couldn't find where we use it so I removed it for now but want to shout it out in case I'm missing something.

Reply:
Does not seem used indeed.